modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,151 @@
+
+* It is recommended to use Redis Session Locking (minimum required version for `php-redis` is `4.1.0`)
+* Enable Redis Session Locking by setting `redis.session.locking_enabled = 1` in `php.ini`
+
+TIP: Please note that Redis Session Locking is not supported in clustered Redis environments.
+
+TIP: If you do not use Redis at all, you do not have to apply changes.
+
+TIP: If your ownCloud deployment is based on the official Docker images or the Univention appliance, these changes have already been applied for you.
+
+**Docs?**
+
+=== Restructured user/group sharing autocompletion
+
+To cope for long user names or additional user information and to provide better overview for users, the user/group sharing autocompletion dropdown has been restructured.
+The available information is now distributed vertically to improve space usage and user experience. Screenshots can be found in the https://github.com/owncloud/core/pull/35397[pull request]. Other ownCloud clients will align with this behavior with the next releases.
+
+=== SWIFT object storage as primary & secondary storage removed
+
+Following the https://doc.owncloud.com/server/admin_manual/release_notes.html#swift-objectstore-deprecation[deprecation announcement] with ownCloud Server 10.0.9, support for primary and secondary storage via the OpenStack SWIFT protocol has been removed. Please get in contact with ownCloud Support if you're still using OpenStack SWIFT and want to upgrade.
+
+=== S3 object storage as secondary storage is now a separate app
+
+The extension to integrate S3 object storages as secondary storages (`files_external_s3`) has been updated, unbundled from ownCloud Server (was previously part of `files_external`) and released to the https://marketplace.owncloud.com/apps/files_external_s3[ownCloud Marketplace]. If you're using S3 external storage mounts, you have to conduct some steps to ensure continuous operation after upgrading to Server 10.3:
+
+* After the upgrade to Server 10.3 has finished successfully, keep the maintenance mode activated and install/enable `files_external_s3` (either manually or via the Market app) as the app is not bundled with ownCloud Server anymore.
+* After enabling `files_external_s3` (`occ app:enable files_external_s3`) it is necessary to run `occ upgrade` again to conduct config migrations. This migration step should run quickly.
+* If users were allowed to configure personal mount points before the upgrade, switch from maintenance mode into single user mode (`occ maintenance:singleuser --on`) and enable the option again by ticking the respective checkbox below "Allow users to mount external storage" (in Admin settings => Storage).
```

Comment:
Click "Amazon S3" Storage ( Sub item of "Enable External Storages")
Maybe Screenshot?